M2/Macaulay2/packages/ConwayPolynomials.m2

```diff
@@ -10,24 +10,28 @@ newPackage(
     	Headline => "a database of Conway polynomials"
     	)
 -- the data comes libflint
-export "conwayPolynomial"
+export {
+    "conwayPolynomial",
+    -- options to set the variable to use
+    "Variable"
```

Comment:
"Variable" is a symbol in the Core package, so just use it -- don't export a new one with the same name.

Reply:
Gotcha! Just changed!